apps/web/src/services/asset-inventory/cloud-service/modules/CloudServiceToolbox.vue

```diff
@@ -140,44 +145,40 @@ const getExcelFields = async (data): Promise<ExcelDataField[]> => {
     return excelField;
 };
 const excelApiQueryHelper = new ApiQueryHelper();
-const getExcelQuery = (data, field) => {
+const getExcelQuery = (data) => {
     excelApiQueryHelper
-        .setFilters(state.cloudServiceFilters)
+        .setFilters(excelState.cloudServiceFilters)
         .addFilter({ k: 'provider', o: '=', v: data.provider })
         .addFilter({ k: 'cloud_service_group', o: '=', v: data.cloud_service_group })
         .addFilter({ k: 'cloud_service_type', o: '=', v: data.cloud_service_type })
-        .setMultiSort([
+        .setMultiSortV2([
             { key: 'provider', desc: false },
             { key: 'cloud_service_group', desc: false },
             { key: 'cloud_service_type', desc: false },
         ]);
-    const fields = field;
-    if (fields) {
-        excelApiQueryHelper.setOnly(...fields.map((d) => d.key));
-    }
     return excelApiQueryHelper.data;
 };
-const getCloudServiceResourcesPayload = (): ExcelPayload => {
-    excelApiQueryHelper.setFilters(state.cloudServiceFilters).setMultiSort([
+const getCloudServiceResourcesPayload = (): ExportOption => {
+    excelApiQueryHelper.setFilters(excelState.cloudServiceFilters).setMultiSortV2([
         { key: 'provider', desc: true },
         { key: 'cloud_service_group', desc: true },
     ]);
-    return {
-        url: '/statistics/topic/cloud-service-resources',
-        param: {
-            query: excelApiQueryHelper.data,
-            labels: cloudServicePageStore.selectedCategories,
-        },
-        fields: CLOUD_SERVICE_RESOURCES_EXCEL_FIELDS,
-        sheet_name: 'Summary',
-        header_message: {
-            title: 'Summary',
+    return ({
+        name: 'Summary',
+        query_type: QueryType.ANALYZE,
+        analyze_query: {
+            ...excelApiQueryHelper.data,
+            group_by: ['provider', 'cloud_service_group', 'cloud_service_type'],
+            fields: {
+                total_count: {
+                    operator: 'count',
+                },
+            },
         },
-        file_name_prefix: FILE_NAME_PREFIX.cloudService,
-    };
+    });
```

Comment:
question: Why did you enclose the return object in `()`? 🧐

Reply:
There's no particular reason. Why did I enter?